lxd/api_cluster.go

```diff
@@ -1044,6 +1045,115 @@ func clusterNodesGet(d *Daemon, r *http.Request) response.Response {
 	return response.SyncResponse(true, urls)
 }
 
+// swagger:operation POST /1.0/cluster/members cluster cluster_members_post
+//
+// Request a join token to add a cluster member.
```

Comment:
The first line is used as a kind of header, so shouldn't use punctuation and be kept short.
Here I'd use `Request a join token`